### PR TITLE
Namespace for the omic allocation prototype

### PR DESCRIPTION
**Pull request by alan**
Namespace for the allocation prototype for Omic

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/omic-allocation-prototype/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: omic-allocation-prototype
```
Comment (alkar) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/omic-allocation-prototype/00-namespace.yaml]: Can you please include the labels and annotations as in the [template](https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespace-resources/00-namespace.yaml)?

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/omic-allocation-prototype/resources/ecr.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"
```
Comment (alkar) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/omic-allocation-prototype/resources/ecr.tf]: Can you please also pin this by changing `?ref=master` to `?ref=1.0`?

**Comment by alan** [comment]
@alkar updated as requested, can you take another look please?

**Comment by alkar** [comment]
@alan are you happy to merge this one?

**Pull request closed without merging**